src/IssueViz.Security/IssueViz.Security.csproj

```diff
@@ -60,13 +60,24 @@
       <CopyToOutputDirectory>Never</CopyToOutputDirectory>
       <Generator>XamlIntelliSenseFileGenerator</Generator>
     </Page>
+    
+    <Compile Update="OpenInIDE\OpenInIDEResources.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>OpenInIDEResources.resx</DependentUpon>
+    </Compile>
 
     <Compile Update="Resources.Designer.cs">
       <DependentUpon>Resources.resx</DependentUpon>
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
     </Compile>
 
+    <EmbeddedResource Update="OpenInIDE\OpenInIDEResources.resx">
```

Comment:
Is there a reason to create a different resource file? I'm not against, just wondering what's our convention.

Reply:
There's no harm in having multiple, and it makes it easier to move/refactor the code. If you have one resource for the assembly it can get very big and it's more awkward to work with (which is what happened with the Integration and Integration.VSIX projects).